README.md

```diff
@@ -170,6 +170,51 @@ import { Agent } from '@newrelic/browser-agent/loaders/agent'
 import { Metrics } from '@newrelic/browser-agent/src/features/metrics'
 ```
 
+## Utilizing Pre-Defined Bundles
+
+The npm package contains our pre-defined bundles within the `/build/` directory. This allows you to use the npm package while still importing a pre-built bundle into your HTML files. The npm package contains all our bundle files but you really only need to host and include one of the agent loaders. The loader will still load all async bundles from the New Relic CDN.
+
+```html
+<!-- Copy the configuration block from the snippet in New Relic -->
+<script>
+  ;window.NREUM||(NREUM={});NREUM.init={ ... };
+
+  ;NREUM.loader_config={ ... };
+  ;NREUM.info={ ... };
+</script>
+
+<!-- Include the lite agent -->
+<script src="/node_modules/@newrelic/browser-agent/build/nr-loader-spa-1.232.1.min.js"></script>
+<!-- Include the pro agent -->
+<script src="/node_modules/@newrelic/browser-agent/build/nr-loader-full-1.232.1.min.js"></script>
+<!-- Include the pro+spa agent -->
+<script src="/node_modules/@newrelic/browser-agent/build/nr-loader-rum-1.232.1.min.js"></script>
+```
+
+The version number in the file name will change depending on the version of the npm package that is installed. It is recommended that you setup your build tooling to copy the correct file from the `/node_modules/@newrelic/browser-agent/build/` directory into a `static` directory where you normally host static files. You can rename the file at the same time to make the `script` element static but remember to manage cache busting for your users.
```

Comment:
Should we just build the agent without the version suffix before shipping NPM so that this is not an issue customers would need to deal with?

Reply:
They could use e.g. `nr-loader-spa-1.x.x.min.js` and that might last them until the next breaking change.

Reply:
This actually might be a good case for generating x.x.x which was removed from the script